src/audio-worklet/free-queue/examples/hello-webassembly/sink-processor.js

```diff
@@ -0,0 +1,38 @@
+import FreeQueue from '../../src/free-queue.js';
+
+const RENDER_QUANTUM = 128
+
+/**
+ * A simple AudioWorkletProcessor node that will act as sink.
+ * It will pull out audio from queue and render it.
+ * 
+ * @class BasicProcessor
+ * @extends AudioWorkletProcessor
+ */
+class SinkProcessor extends AudioWorkletProcessor {
+  constructor(options) {
+    super();
+    // Initialize the Queue.
+    // We have to set prototype of queue, because queue object is passed
+    // using structured cloning algorithm.
+    // See - 
+    // https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm
+    this.queue = options.processorOptions.queue;
+    Object.setPrototypeOf(this.queue, FreeQueue.prototype);
+  }
+
+  process(inputs, outputs) {
+    const output = outputs[0];
```

Comment:
So this means it only fills up a single channel (mono). Why can't we do the stereo?

Reply:
This library just acts as queue to share data between 2 threads.
So, I think it's up to user how they use it.

inputs and outputs are array of arrays of arrays (3 dimensional arrays)

```js
outpus: [
  output_0: [
    channel_0: [0, 0, 0.5, 0.6....],
    channel_1: [0, 0, 0.5, 0.6....]
  ],
  output_1: [
    channel_0: [0, 0, 0.5, 0.6....],
    channel_1: [0, 0, 0.5, 0.6....],
  ]
]

```
So, we are technically pushing data into 2 channels of 0th output in this case